examples/moveit.py

```diff
@@ -49,23 +69,45 @@ def main():
     screen.blit(background, (0, 0))
 
     objects = []
+    p = GameObject(player, 10, 3)
     for x in range(10):
-        o = GameObject(player, x * 40, x)
+        o = GameObject(entity, x * 40, x)
         objects.append(o)
 
+    pg.display.set_caption("Move It!")
+
+    # This is a simple event handler that enables player input.
     while True:
-        for event in pg.event.get():
-            if event.type in (pg.QUIT, pg.KEYDOWN):
-                return
+
+        #Get all keys currently pressed, and move when an arrow key is held.
+        keys = pg.key.get_pressed()
+        if keys[pg.K_UP]:
+            p.move(up=True)
+        if keys[pg.K_DOWN]:
+            p.move(down=True)
+        if keys[pg.K_LEFT]:
+            p.move(left=True)
+        if keys[pg.K_RIGHT]:
+            p.move(right=True)
+
+        #Draw the background
+        screen.blit(background, (0, 0))
+        for e in pg.event.get():
+
+            # quit upon screen exit
+            if e.type == pg.QUIT:
+                exit()
```

Comment:
It'd still be better to `return` here IMHO.... `exit` is not the best of functions to use to exit applications